normandy/base/utils.py

```diff
@@ -18,3 +18,12 @@ def urlparams(url, fragment=None, **kwargs):
         urlencode(query, doseq=True),
         fragment if fragment is not None else parsed.fragment
     ))
+
+
+class Whatever(object):
```

Comment:
Input has something like this. It is useful for tests. Input put it in `fjord/base/tests/__init__.py`, but we don't have a `normandy/base/tests/` yet, so I put it here.